drivers/avr/analog.c

```diff
@@ -21,17 +21,55 @@
 #include <stdint.h>
 #include "analog.h"
 
-static uint8_t aref = (1 << REFS0);  // default to AREF = Vcc
+static uint8_t aref = ADC_REF_POWER;  // default to AREF = Vcc
 
 void analogReference(uint8_t mode) { aref = mode & 0xC0; }
 
+uint8_t pinToADCIndex(uint8_t pin){
```

Comment:
It looks like this change is breaking something in `convert/usb_usb/ble`.

Reply:
Specifically, BLE also uses analog.c, but there is some sort of incompatibility here.  Mostly, because I think that the board in question doesn't have the config_common.h file included that includes these definitions. 
```
Compiling: drivers/avr/analog.c                                                                    drivers/avr/analog.c: In function 'pinToADCIndex':
drivers/avr/analog.c:31:7: error: 'B4' undeclared (first use in this function); did you mean 'PB4'?
       B4, 8,
       ^~
       PB4
drivers/avr/analog.c:31:7: note: each undeclared identifier is reported only once for each function it appears in
drivers/avr/analog.c:32:7: error: 'B5' undeclared (first use in this function); did you mean 'PB5'?
       B5, 7,
       ^~
       PB5
```

Reply:
I can't find where the BLE keyboard calls or builds the analog file. I should be able to reproduce the problem with "make converter/usb_usb/ble"?